sql/migrate/V006__DS13.sql

```diff
@@ -74,6 +74,19 @@ CREATE TABLE IF NOT EXISTS `database`.`store_schematics` (
 ENGINE = InnoDB;
 
 
+-- -----------------------------------------------------
+-- Table structure for table `role_time`
+-- -----------------------------------------------------
+
+DROP TABLE IF EXISTS `role_time`;
+
+CREATE TABLE `role_time` (
+  `ckey` VARCHAR(32) NOT NULL ,
+  `job` VARCHAR(32) NOT NULL ,
+  `minutes` INT UNSIGNED NOT NULL,
+  PRIMARY KEY (`ckey`, `job`))
+ENGINE = InnoDB;
+
```

Comment:
This should really be moved to a new Migrate file, these are intended to be run sequentially and never updated. They're like github commits for the database structure.